pkg/auth/scram.go

```diff
@@ -357,6 +357,8 @@ func (s *Scram) getCBindInputString() string {
 		switch s.params.cbMechanism {
```

Comment:
Please, add a brief description for this PR to the [`main / unreleased`](https://github.com/ortuman/jackal/blob/main/CHANGELOG.md#jackal---main--unreleased) CHANGELOG.md section